### PR TITLE
[MTS][87807824] Replace Foreman with Gulp

### PR DESCRIPTION
**Pull request by michaelstalker**
https://www.pivotaltracker.com/story/show/87766816

This is a move that will help standardize our way of compiling CoffeeScript to JavaScript.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+gulp.task('build', function() {
+  gulp.run('coffee');
+});
+
+gulp.task('default', function () {
+  gulp.run('build');
+
+  gulp.watch('src/**/*.coffee', function (event) {
+    gulp.run('coffee');
```
Comment (c0) [inline gulpfile.js:29]: `gulp.run` is deprecated. It's recommended to either switch to passing them in as an array or use `run-sequence`. For tasks that are order specific, I've had better success with [`run-sequence`](https://github.com/rentpath/tautline/blob/master/gulp/tasks/build.js#L5)

Comment (Teapane) [inline gulpfile.js:29]: Should we address this in [my-places](https://github.com/rentpath/my-places.js/blob/dev/gulpfile.js#L43) as well then?

Comment (c0) [inline gulpfile.js:29]: :+1:

right now gulp gives a warning. I'm not sure what their roadmap is for removing it.

Comment (Teapane) [inline gulpfile.js:29]: Cool. I'll take a look after this map bug.

**Comment by c0** [comment]
After speaking with @michaelstalker, `gulp.run` will be addressed in another PR/story.

**Comment by Teapane** [comment]
@c0 & @michaelstalker I am looking at `gulp.run` now. However, [runSequence](https://www.npmjs.com/package/run-sequence?) seems to also be a temporary fix...Thoughts?

**Comment by michaelstalker** [comment]
https://www.pivotaltracker.com/story/show/87973084
https://www.pivotaltracker.com/story/show/87972980
:boom:

**Pull request merged**